src/app/app.component.html

```diff
@@ -1,16 +1,16 @@
 <header class="header">
-    <div class="logo">
+    <div class="logo" (click)="handleGoHome()">
         <span class="logo-left">Cinema</span><span class="logo-right">tics</span>
     </div>
     <nav class="nav">
         <a routerLink="/admin" routerLinkActive="active">Admin</a>
+        <a routerLink="" routerLinkActive="active">General</a>
         <a routerLink="/auth" routerLinkActive="active">Auth</a>
     </nav>
 </header>
```

Comment:
Usually, header and footer are separate components

Reply:
done